submissions/submissions.md

```diff
@@ -0,0 +1,10 @@
+Explain why Rails exists and what it's used for.
+Rails is a web framework that provides a server back end for for web based applications.  It consists of a server, controllers, and views.  The server takes in client initiated requests and sends responses to the clients.  It parses the requests & sends them to the appropriate controller.  The controller sends the request to the appropriate view.  The view processes the requests and interfaces with a database if needed and sends a response to the controller, which compiles the response into html and sends it back to the server which passes it back to the clients.  Not certain who determines if there is an error, assuming it is the controoler or view.
```

Comment:
A 'view' is so named because it's something for the user to view - it's intimately related to the application's UI.